src/corehost/cli/deps_format.h

```diff
@@ -116,6 +114,8 @@ class deps_json_t
     bool m_valid;
 
     pal::string_t m_deps_file;
+
+    json_parser_t m_json;
```

Comment:
Why do we need to hang on to this? Do we have pointers to its buffer after parsing? And can you add a comment about it here?

Reply:
That used to be the case, yes, and ended up forgetting to remove this.  Will be declared in `deps_json_t::load()`.